packages/terra-popup/src/_PopupContent.jsx

```diff
@@ -237,6 +237,8 @@ class PopupContent extends React.Component {
             onResize={this.handleOnResize}
             refCallback={refCallback}
             role={popupContentRole || null}
+            onKeyDown={event => event.stopPropagation()} // Added for Popup support in terra-compact-interactive-list. As focus trap doesn't stop key press event propagation to the CIL cell, it interferes with cell key press event handler.
```

Comment:
You should create a function will a useCallback hook for memoization to prevent unnecessary rerenders.  Same comment for your new onFocus function.

Reply:
Fixed here: https://github.com/cerner/terra-framework/pull/2149/commits/0dc66ba494b2880e5c257785e1d85fabc43c1632